assets/src/blocks/PostsList/PostsListBlock.js

```diff
@@ -0,0 +1,68 @@
+export const POSTS_LIST_BLOCK = 'planet4-blocks/posts-list-block';
+
+export const registerPostsListBlock = () => {
+  const {registerBlockVariation} = wp.blocks;
+  const {__} = wp.i18n;
+
+  return registerBlockVariation('core/query', {
+    name: POSTS_LIST_BLOCK,
+    title: 'P4 Posts List Block',
```

Comment:
Don't we need to translate this text?

Reply:
I think we don't offer translation for our block names, just to make things less confusing on Handbook documentation.

Reply:
Can't we just use "Posts List" as a name?

Reply:
I changed the name because WP offers a variation of the Query Loop which is also called Posts List, so we may need to change ours or add `Block` at the end to differentiate the two.

<img width="334" alt="Screenshot 2023-11-02 at 09 42 04" src="https://github.com/greenpeace/planet4-master-theme/assets/38656549/7186b92a-869f-401f-8a5b-83e7d9b7b9c3">